### DISCUSSION TITLE
Updating a required relation?

### DISCUSSION THREAD

**Discussion by rasberrylasagna** [prisma/prisma#7468] [Q&A] [answered]
Hey guys, just a quick question, is it normal behavior that I'm unable to update a one-to-one relationship on the required side?

I'm sure that the update value for the relation is correct / unused. When I do, I get this error
`The change you are trying to make would violate the required relation...`

This may be a complete oversight but thanks in advance for the help!

**Comment by janpio** [comment, marked as answer]
See this bug: https://github.com/prisma/prisma/issues/2057

**Reply by rasberrylasagna** [reply]
Ah, so this affects updates as well? Understood. Thanks for the help Jan! :)

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
